apps/server/src/modules/learnroom/uc/lesson-copy.uc.ts

```diff
@@ -37,8 +37,8 @@ export class LessonCopyUC {
 		const copyName = this.copyHelperService.deriveCopyName(originalLesson.name, existingNames);
```

Comment:
please use the logic for the copyname into the copyservice as well. 
And im pretty sure line 31-33 can just be removed.

The authorisation checks can stay here. Again, use course copy uc and service as blueprint.

Reply:
Moving copyname to copyService would require more changes because copyLesson() is also used by two other classes. Furthermore, copyLesson() already provides an option not to specify a copy name, which however produces a different result than the one currently used. If this refactor is badly needed, create a jira ticket for it as fixing this code was not part of this PR.

I removed CourseRepo from there and make the code a bit clearer:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/b54ff6c0ecd2eb23d533490f884d4de2be5f1a59/apps/server/src/modules/learnroom/uc/lesson-copy.uc.ts#L29-L38

Reply:
thats fair, please create a ticket for it.

Reply:
https://ticketsystem.dbildungscloud.de/browse/BC-3196